railties/test/generators/app_generator_test.rb

```diff
@@ -441,6 +441,12 @@ def test_application_name_is_normalized_in_config
     assert_file "MyWebSite/config/database.yml", /my_web_site_production/
   end
 
+  def test_html_language_attribute
+    run_generator
+
+    assert_file "app/views/layouts/application.html.erb", /<html lang="en">/
```

Comment:
Am I correct to assume that this generator spits out a static `lang` attribute?

Why not spit out the full ERB to determine the `lang` at runtime?

(I work on some multi-lang content sites in Rails, we do this.)

The `I18n` locale is probably always going to be `:en` when this generator runs right? Since it runs on `rails new`?

Even if a site isn't multi-language.. Having the `lang` attribute influenced by a single locale setting is probably nicer?

Lots of ifs and maybes in my comment here 😁😁